stix2/sources/__init__.py

```diff
@@ -146,14 +146,14 @@ class DataSink(object):
     different sink components.
 
     Attributes:
-        id (str): A unique UUIDv4 to identify this DataSink.
+        id_ (str): A unique UUIDv4 to identify this DataSink.
```

Comment:
Is something (pycodestyle, flake8, etc.) complaining about our use these names (`id` and `filter`)? Even though we are shadowing builtins, they aren't builtins we're actually using, so IMO we shouldn't use trailing underscores (I regret having done this in python-stix). If we're using *keywords* (like `class`, `import`, `from`, `for`), then we'll need to use the trailing underscore to prevent syntax errors.

Reply:
Alright. I thought it as an oversight, I'll revert that.